roles/install-nfs/tasks/main.yaml

```diff
@@ -0,0 +1,88 @@
+---
+- name: Install prereq & nfs client packages
+  yum:
+    name: "{{ packages }}"
+    state: present
+  vars:
+    packages:
+      - wget
```

Comment:
where is this used `wget`?

Reply:
@mkumatag 
It was used to download yq, it's no longer needed since we updated the scipt. Have removed it.